bot/exts/utilities/wolfram.py

```diff
@@ -303,4 +303,7 @@ async def wolfram_short_command(self, ctx: Context, *, query: str) -> None:
 
 async def setup(bot: Bot) -> None:
     """Load the Wolfram cog."""
+    if not Wolfram.key:
```

Comment:
`Wolfram` is the cog, not the constants config. This causes an error.

Reply:
My bad, I hadn't still tested this yet, and I was trying to get something in a PR before I left the house.
5ba5275ab343ec36f2a2ab99f7d5a6dd2f4db8f6